### PR TITLE
 enable specifying enable/ensure when managing services

### PR DESCRIPTION
**Pull request by dutsmiller**
## Description
This allows for changing the default setting for service enable/ensure when manage_service is set to true.

## Motivation and Context
We are using packer to build AMIs and have a need for the sensu client service to be set to:
    ensure => stopped
    enable => true
The reason for this is we do not want the packer instance to kick off the monitoring client while our AMI is being built, however we do want the service to start automatically whenever the AMI is eventually launched.

### CODE REVIEW

**Comment by ghoneycutt** [comment]
Thanks for the PR!!

Could you please add spec tests for the new parameters?

**Comment by dutsmiller** [comment]
Hmm, sadly, I can't say I've ever used rspec.  I'll have to take a look.

**Comment by dutsmiller** [comment]
@ghoneycutt before I go any farther, is that what you're looking for?  (sorry, don't really know how to test rspec code).

File: spec/classes/sensu_client_spec.rb [hunk 1]
```diff
@@ -232,14 +232,68 @@
...
+      end # with enable=false ensure=stopped hasrestart=true
+
+      context 'with enable=false ensure=stopped hasrestart=false' do
+        let(:params) { { :client => true, :client_service_enable => false, :client_service_ensure => 'stopped', :hasrestart => false } }
+        it { should contain_service('sensu-client').with(
+          :ensure     => 'stopped',
+          :enable     => false,
+          :hasrestart => false
+        ) }
+      end # with enable=false ensure=stopped hasrestart=false
```
Comment (ghoneycutt) [inline spec/classes/sensu_client_spec.rb]: this is great!

Comment (dutsmiller) [reply spec/classes/sensu_client_spec.rb]: I don't see any server service spec code to update, so is my work here done?

**Comment by dutsmiller** [comment]
Just checking back in, any chance of cutting a new release?

**Comment by ghoneycutt** [comment]
Solves #888

**Comment by ghoneycutt** [comment]
Rebased in PR #889

**Comment by ghoneycutt** [comment]
Released in v2.51.0

**Comment by dutsmiller** [comment]
awesome, thanks for the merge!

**Pull request closed without merging**